test/e2e/Deploy.t.sol

```diff
@@ -432,49 +432,29 @@ abstract contract CommonDeploymentTest is HaiTest, Deploy {
+    // NOTE: (deployment)block.timestamp + 1 days (cannot be tested)
+    // assertEq(tokenDistributor.claimPeriodStart(), _tokenDistributorParams.claimPeriodStart);
     assertEq(tokenDistributor.claimPeriodEnd(), _tokenDistributorParams.claimPeriodEnd);
   }
 
+  function test_Delegated_OP() public {
+    assertEq(ERC20Votes(OP_OPTIMISM).delegates(address(collateralJoin[OP])), address(haiDelegatee));
+  }
+
   function _test_Authorizations(address _target, bool _permission) internal {
-    // base contracts
-    assertEq(safeEngine.authorizedAccounts(_target), _permission);
-    assertEq(oracleRelayer.authorizedAccounts(_target), _permission);
-    assertEq(taxCollector.authorizedAccounts(_target), _permission);
-    assertEq(stabilityFeeTreasury.authorizedAccounts(_target), _permission);
-    assertEq(liquidationEngine.authorizedAccounts(_target), _permission);
-    assertEq(accountingEngine.authorizedAccounts(_target), _permission);
-    assertEq(surplusAuctionHouse.authorizedAccounts(_target), _permission);
-    assertEq(debtAuctionHouse.authorizedAccounts(_target), _permission);
-
-    // settlement
-    assertEq(globalSettlement.authorizedAccounts(_target), _permission);
-    assertEq(postSettlementSurplusAuctionHouse.authorizedAccounts(_target), _permission);
-    assertEq(settlementSurplusAuctioneer.authorizedAccounts(_target), _permission);
-
-    // factories
-    assertEq(chainlinkRelayerFactory.authorizedAccounts(_target), _permission);
-    assertEq(uniV3RelayerFactory.authorizedAccounts(_target), _permission);
-    assertEq(denominatedOracleFactory.authorizedAccounts(_target), _permission);
-    assertEq(delayedOracleFactory.authorizedAccounts(_target), _permission);
-
-    assertEq(collateralJoinFactory.authorizedAccounts(_target), _permission);
-    assertEq(collateralAuctionHouseFactory.authorizedAccounts(_target), _permission);
-
-    // tokens
-    assertEq(systemCoin.authorizedAccounts(_target), _permission);
-    assertEq(protocolToken.authorizedAccounts(_target), _permission);
-
-    // token adapters
-    assertEq(coinJoin.authorizedAccounts(_target), _permission);
-
-    // jobs
-    assertEq(accountingJob.authorizedAccounts(_target), _permission);
-    assertEq(liquidationJob.authorizedAccounts(_target), _permission);
-    assertEq(oracleJob.authorizedAccounts(_target), _permission);
-
-    // token distributor
-    assertEq(tokenDistributor.authorizedAccounts(_target), _permission);
+    if (_permission) {
+      _toAllAuthorizableContracts(_fn_HasAuthorizations, _target);
+    } else {
+      _toAllAuthorizableContracts(_fn_NoAuthorizations, _target);
+    }
+  }
+
+  function _fn_NoAuthorizations(IAuthorizable _contract, address _target) internal {
+    assertEq(_contract.authorizedAccounts(_target), false);
+  }
+
+  function _fn_HasAuthorizations(IAuthorizable _contract, address _target) internal {
+    assertEq(_contract.authorizedAccounts(_target), true);
   }
```

Comment:
I suggest moving these functions and `_test_Authorizations` to `Common.t.sol:Common`, imitating `Common.s.sol:Common`.

Reply:
`assertEq` is not available on `Common.s.sol`

Reply:
I mentioned `Common.t.sol:Common`, which inherits `HaiTest`.

Reply:
oh, emmm, only 2 methods are declared there `_setCollateralPrice` and `_collectFees`, seems very odd to move them there, as they're declared and implemented on the same part of the code, have hard feelings about it?

Reply:
I think that way the code could look tidier, as those methods seem more general than ad-hoc, but no hard feelings about it. 😆